dvc/fs/http.py

```diff
@@ -114,7 +120,7 @@ async def get_client(self, **kwargs):
             total=None,
             connect=self.REQUEST_TIMEOUT,
             sock_connect=self.REQUEST_TIMEOUT,
-            sock_read=None,
+            sock_read=self.REQUEST_TIMEOUT,
```

Comment:
I'm assuming that setting the `sock_read=self.REQUEST_TIMEOUT` (60s) will not create any issues: the default buffer size is 2**16,  

See `ClientSession._request`:
https://github.com/aio-libs/aiohttp/blob/f5ff95efe278c470a2ff65cabbb5f5f08ba07416/aiohttp/client.py#L511-L519

https://github.com/aio-libs/aiohttp/blob/f5ff95efe278c470a2ff65cabbb5f5f08ba07416/aiohttp/client_proto.py#L17

https://github.com/aio-libs/aiohttp/blob/f5ff95efe278c470a2ff65cabbb5f5f08ba07416/aiohttp/streams.py#L111